src/_includes/scripts/index.js

```diff
@@ -0,0 +1,134 @@
+function init() {
+  var analyticsCookieName = "modernising_cookie_policy";
+  var gaCookieValue = getCookie(analyticsCookieName);
+  var gaCookieSettings = document.querySelector(
+    'input[name="ga-preferences"]:checked'
+  );
+  var isAnalyticsCookieSet = gaCookieValue !== null;
+  document.addEventListener("click", (e) => {
+    if (e.target.name === "cookies") {
+      if (e.target.value === "update") {
+        var newCookieSetting = document.querySelector(
+          'input[name="ga-preferences"]:checked'
+        );
+        if (newCookieSetting !== null && newCookieSetting.value !== null) {
+          setCookie(analyticsCookieName, newCookieSetting.value === "true");
+        }
+        document.location.href = "/";
+      }
+      if (e.target.value === "accept") {
+        setCookie(analyticsCookieName, true);
+        toggleCookieBanner(true);
+        toggleCookieBannerConfirm(false);
+      }
+      if (e.target.value === "reject") {
+        setCookie(analyticsCookieName, false);
+        toggleCookieBanner(true);
+        toggleCookieBannerReject(false);
+      }
+      if (e.target.value === "hide") {
+        toggleCookieBannerConfirm(true);
+        toggleCookieBannerReject(true);
+      }
+    }
+
+    if (isAnalyticsCookieSet) {
+      if (e.target) {
+        if (
+          e.target.getAttribute("href") &&
+          e.target.getAttribute("href").indexOf("http") === 0
+        ) {
+          window.gtag("event", "click", {
+            event_category: "outbound",
+            event_label: e.target.getAttribute("href"),
+          });
+        }
+      }
+    }
+  });
+  if (isAnalyticsCookieSet) {
+    toggleCookieBanner(true);
+    toggleCookieBannerConfirm(true);
+    toggleCookieBannerReject(true);
+    if (gaCookieValue == "true") {
+      setupGoogleAnalytics("G-70EPL8ZB7Y");
+    }
+  }
+
+  if (location.pathname === "/view-cookies/") {
+    if (gaCookieSettings === null || gaCookieValue == "false") {
+      document.querySelector(
```

Comment:
Does it streamline this to put the selector in a string and run the querySelector outside the if.

Reply:
Will look again tomorrow at this, think there is a better way but needing the true/false value of the radio is an issue.